ngraph/core/include/ngraph/pass/low_latency.hpp

```diff
@@ -40,9 +38,20 @@ namespace ngraph
  *                                                              \
  *                                                               -> Result ] -> back_edge_1
  *
- *  It is recommended to use this transformation in conjunction with the UnrollTI transformation.
- *  After applying both of these transformations, you will have a network that allows you to infer
- *  step by step, the state will store between inferences.
+ *  It is recommended to use this transformation in conjunction with the Reshape feature to set sequence
+ *  dimension to 1 and with the UnrollTensorIterator transformation.
+ *  For convenience, we have already enabled the unconditional execution of the UnrollTensorIterator
+ *  transformation when using the LowLatency transformation for CPU, GNA plugins, no action is required here.
+ *  After applying both of these transformations, the resulting network can be inferred step by step,
+ *  the states will store between inferences.
+ *
+ *    An illustrative example, not real API:
+ *
+ *    network->reshape(...) // Set sequence dimension to 1, recalculating shapes. Optional, depends on the network.
+ *    LowLatency(network)   // Applying LowLatency and UnrollTensorIterator transformations.
+ *    network->infer (...)  // Calculating new values for states.
+ *    // All states are stored between inferences via Assign, ReadValue layers.
+ *    network->infer (...)  // Using stored states, calculating new values for states.
```

Comment:
it's IE code, why is it in ngraph headers?

Reply:
the implementation is here openvino/ngraph/core/src/pass/low_latency.cpp, ngraph code.
INFERENCE_ENGINE_API_CPP(void) LowLatency(InferenceEngine::CNNNetwork& network); is a wrapper.

Reply:
I understand. The point is the code snippets you provided should be moved to wrapper docs.

Reply:
*    An illustrative example, not real API
This is not a real code example, just an explanation of why the transformation is needed and what does it allow to do

Reply:
I agree that we shouldn't add IE dependent snippet to nGraph code.

Reply:
fixed